packages/headless/src/api/search/search-api-client.test.ts

```diff
@@ -364,104 +362,101 @@ describe('search api client', () => {
           },
         });
       });
+    });
 
-      it(`when calling SearchAPIClient.recommendations
+    it(`when calling SearchAPIClient.recommendations
```

Comment:
Changes from here onward are just indentation, due to moving two tests out of a `describe` block dedicated to facetSearch.